client/my-sites/patterns/types.ts

```diff
@@ -21,14 +21,27 @@ type CategoryBase = {
 
 export type CategorySnakeCase = CategoryBase & {
 	page_pattern_count: number;
-	regular_cattern_count: number;
+	page_preview_pattern: Pattern | null;
+	regular_pattern_count: number;
+	regular_preview_pattern: Pattern | null;
 };
 
 export type Category = CategoryBase & {
 	pagePatternCount: number;
+	pagePreviewPattern: Pattern | null;
 	regularPatternCount: number;
+	regularPreviewPattern: Pattern | null;
 };
 
+export type CategoryGalleryProps = {
```

Comment:
We don't use it outside
```suggestion
type CategoryGalleryProps = {
```